PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,6 +2,7 @@
 
 * Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
 * All new code requires tests to ensure against regressions
+* If you add/remove/modify any event please update the [documentation](docs/events.md)
```

Comment:
I worry this seems prone to the human error of forgetting.

I can't think of an easy way to automatically or programmatically keep this updated though  :-(

Reply:
Yup I agree... Unless we move to a fixed schema for the events and use it to generate the docs I can't see an easy way to ensure that this doc is up to date... And this seems like too much work